Documentation/ApiOverview/Localization/TranslationServer/Crowdin/Bestpractice.rst

```diff
@@ -0,0 +1,278 @@
+
+TYPO3 has already organized projects and files in a way that follows Crowdin's
+own recommendations. However, as a translator, there will be settings and working
+methods that you can adapt to your own needs. This gives you flexibility and the
+opportunity to work in the way that suits you best – whether you prefer to translate
+directly in Crowdin, whether you use EXT:Crowdin, or if you want to work offline
+and upload the files later.
+
+..  _crowdin-bestpractice-initial:
+
+Initial considerations
+----------------------
+
+Before you start translating a project, there are some initial considerations that
+may be good to consider. For example, start with a project that has a smaller
+number of words and strings. This makes the work more manageable and increases
+the joy of finishing a project, so you avoid getting completely worn out. It's
+important to choose a project that motivates you so that you stay engaged.
+
+A good idea is to start by translating a single file in a project. The different
+files are often used in different parts of an extension, and it is easier to
+understand the context if you work with individual files. This will help you
+understand how the text is used and ensure that the translation is appropriate
+for the situation.
+
+Choose a project that you think is already used in your language area, or a project
+that you are going to work on and want to give your customers/users access to in
+their own language. Alternatively, you can choose one of the projects that in
+TYPO3's Extension Repository have the most likes and are thus also the most popular.
+Popular projects typically have a lot of users, so your contribution will have a
+big impact.
+
+If you want to get started with the TYPO3 CMS project, there are typically many
+words (>57.000 and counting) and strings that need to be translated. Here, it is
+especially important to have a good plan from the start: Start with files in the
+main branch – then you know that you will hit the most users. We still hold previous
+(E)LTS versions in but unless you have massive energy, leave these files.
+
+And follow the recommendations above, work in a structured way through files and
+projects. It can be an advantage to set intermediate goals along the way, so that
+you stay motivated even if the project is big.
+
+To wrap it up:
+
+*   Start with a small project with a few words and strings to make the work manageable.
+*   Choose a project that motivates you to keep your spirits up.
+*   Feel free to translate one file at a time to get a better understanding of the context.
+*   Choose projects that are already in use in your language area or that you will be working on yourself.
+*   Consider tackling popular projects – your contribution will have a greater impact.
+*   If you are translating TYPO3 CMS, start with files in the main branch to reach the most users.
+*   Work in a structured way and preferably set sub-goals to maintain motivation in larger projects.
```

Comment:
Shall we add a note that for TYPO3 core translation, it would make sense to start with the Glossary?